viewing_party/main.py

```diff
@@ -0,0 +1,165 @@
+def create_movie(title, genre, rating):
+      # if all three parameters are truthy, return a dict with key val pairs
+       #return a dictionary with 3 keys title, genre, rating 
+    #else title, genre, or rating is falsey
+        #return None
+    movie_dict = {}
+    movie_args = []
+    movie_args.append(title)
+    movie_args.append(genre)
+    movie_args.append(rating)
+    for i in movie_args:
+        if not i:
+            return None
+            
+    
+    movie_dict['title'] = movie_args[0]
+    movie_dict['genre'] = movie_args[1]
+    movie_dict['rating'] = movie_args[2]
```

Comment:
Creating a list of the arguments was a little too much work for this problem. In doing so, you've obscured what the parameters are. Consider this refactor:
```suggestion
    movie_dict['title'] = title
    movie_dict['genre'] = genre
    movie_dict['rating'] = rating
```

There certainly will be times that you'll want to put function arguments into a list, so it's good to see you practiced with this type of manipulation.